pom.xml

```diff
@@ -91,8 +91,8 @@
         <scala.xml.version>1.3.0</scala.xml.version>
         <scalatest.version>3.2.9</scalatest.version>
         <scala.binary.flink.version>_${scala.binary.version}</scala.binary.flink.version>
-
-        <flink.version>1.14.4</flink.version>
+        <flink.connector.version>3.2.0-1.18</flink.connector.version>
+        <flink.version>1.18.1</flink.version>
         <flink.shaded.version>1.8.1</flink.shaded.version>
```

Comment:
I agree to upgrade flink 1.18, but I am not sure whether it will lead to other problems, which need to be evaluated @wolfboys

Reply:
We have e2e guarded.